### ISSUE TITLE
Only components tests are instrumented, not e2e tests

### ISSUE THREAD

**Issue by ogous** [MenaiAla/nextjs-cypress-code-coverage#2]
Hey, tried your solutions, worked fine for components tests. But when I would like to try for e2e after runing dev server `npm run dev` e2e test is not instrumented so there is no window.__coverage__ object and cypress warns about it.

<img width="321" alt="Screenshot 2024-07-06 at 19 09 57" src="https://github.com/alamenai/nextjs-cypress-code-coverage/assets/47118973/520f2848-7952-4973-829b-d190d41a6725">

Am I missing something?
<br/>
<hr/>

<details><summary>This repo is using Opire - what does it mean? 👇</summary><br/>💸Everyone can add rewards for this issue commenting <code>/reward 100</code> (replace <code>100</code> with the amount)<br/>💪 If someone starts working on this issue to earn the rewards, they can comment <code>/try</code> to let everyone know!<br/>🙌 And when they open the PR, they can comment <code>/claim #2</code> either in the PR description or in a PR's comment<br/><br/>👀 Also, everyone can tip any user commenting <code>/tip 20 @ogous</code> (replace <code>20</code> with the amount, and <code>@ogous</code> with the user to tip)<br/><br/>If you want to learn more, go to our <a href="https://docs.opire.dev">documentation</a></details>

**Comment by filipe-jsales** [comment]
I'm facing the same issue here, the coverage lines says 100% but with 0/0 all
![image](https://github.com/user-attachments/assets/a650bd45-ea83-4eb4-8b33-330acda423db)

**Comment by DmitryOlkhovoi** [comment]
Yeah guys. To do that we again have the same problem - we need to instrument the npm run dev. 
In her article she runs server and then component tests lol. 
Where components are imported in Cypress test cases and with babel. So coverage works. But when you run server without babel it doesn't work.
